index.html

```diff
@@ -132,12 +152,127 @@ <h3 class="project_name">
 </ul>
 <button class="btn1" type="submit">See Project</button>
 </div>
+</div>
 </section>
-  
+</main> -->
+<main id="work_section" class="mobile-nav">
```

Comment:
- [ ] You did a great job 👍 But need to fix some padding here to make your work looks like the one in Figma. 
- Please, reduce the **padding-right / padding-left** in .desk-1, .desk-2, .desk-3, .desk-4
- Please, You can also try to reduce the **margin** in .btn1 
- Kindly check if you can reduce or delete the **width** in .side_content

![r2](https://user-images.githubusercontent.com/63972670/161390040-3d3e0c9c-7034-4a0c-8b11-41c0120ea56a.JPG)